CMakeLists.txt

```diff
@@ -304,7 +304,7 @@ endif()
 # library. If not building a shared library ``-fPIC`` isn't needed and would add
 # unnecessary overhead.
 if (BUILD_LIBZ3_SHARED)
-  if (NOT MSVC)
+  if (NOT (MSVC OR MINGW))
```

Comment:
@martin-neuhaeusser This LGTM. However please add a small comment here that says something similar to ``mingw64`` always generates position independent code so passing ``-fPIC`` is not necessary. I know this message currently is the commit message but this will get lost if someone touches the code here and it may not be obvious to a reader why MinGW doesn't need ``-fPIC``.